protocol/nats_jetstream/v2/message.go

```diff
@@ -25,7 +36,13 @@ type Message struct {
 // NewMessage wraps an *nats.Msg in a binding.Message.
 // The returned message *can* be read several times safely
 func NewMessage(msg *nats.Msg) *Message {
-	return &Message{Msg: msg, encoding: binding.EncodingStructured}
+	encoding := binding.EncodingStructured
```

Comment:
can we update the doc string then that the default returned will assume structured encoding?